collector/lib/HostHeuristics.cpp

```diff
@@ -100,11 +100,28 @@ class ARM64Heuristic : public Heuristic {
   }
 };
 
+class PowerHeuristic : public Heuristic {
+ public:
+  void Process(HostInfo& host, const CollectorConfig& config, HostConfig* hconfig) const {
+    auto k = host.GetKernelVersion();
+
+    if (k.machine != "ppc64le") {
+      return;
+    }
+
+    if (host.IsRHEL86()) {
+      CLOG(INFO) << "CORE_BPF is not available, switching to EBPF collection method.";
```

Comment:
We can probably refine this message a bit to mirror the one on line 86
```suggestion
      CLOG(INFO) << "RHEL 8.6 on ppc64le does not support CORE_BPF, switching to eBPF collection method.";
```